graylog2-server/src/main/java/org/graylog2/cluster/nodes/DataNodeDto.java

```diff
@@ -63,6 +65,11 @@ public abstract class DataNodeDto extends NodeDto {
     @JsonProperty(FIELD_CERT_VALID_UNTIL)
     public abstract Date getCertValidUntil();
 
+    @jakarta.annotation.Nullable
```

Comment:
I think the other `@Nullable`s should also be changed here

Reply:
from javax to jakarta, i mean

Reply:
Thanks, fixed!